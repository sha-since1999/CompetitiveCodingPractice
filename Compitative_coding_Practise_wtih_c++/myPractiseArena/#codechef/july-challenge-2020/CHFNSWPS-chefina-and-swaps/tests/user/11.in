8
5
1 3 3 15 15
1 8 8 12 12
4
3 3 15 15
8 8 12 12
4
3 3 4 4
8 8 12 12
11
3 3 4 4 5 5 6 6 7 7 7 
8 8 12 12 23 23 23 23 7 33 33
4
1 2 4 4
1 1 1 2
4
1 1 98 98
99 99 100 100
10
6 6 6 6 6 6 7 7 9 9 
3 3 4 4 5 5 8 8 10 10
7
2 2 100 105 105 98 98
2 2 100 100 100 1 1
