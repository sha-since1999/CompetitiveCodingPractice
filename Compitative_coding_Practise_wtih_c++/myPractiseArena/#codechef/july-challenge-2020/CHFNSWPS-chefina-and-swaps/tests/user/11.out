4
9
7
24
1
3
24
3